cas-server-webapp/src/main/webapp/WEB-INF/deployerConfigContext.xml

```diff
@@ -61,7 +61,7 @@
                    | set its name to something other than its default name (typically the simple class name).
                    -->
                 <entry key-ref="proxyAuthenticationHandler" value-ref="principalResolver" />
-                <entry key-ref="ldapPasswordHandler" value-ref="principalResolver"/>
+                <entry key-ref="ldapPasswordHandler" value="#{null}"/>
```

Comment:
Why removing the principalResolver here ?

Reply:
Because iirc if i leave it, something doesnt work.. as i said in the first comment:

> Removing the principalResolver from ldapPasswordHandler is awkward but seems required per https://apereo.github.io/cas/4.2.x/installation/Configuring-Principal-Resolution.html#principalresolver-vs-authenticationhandler

Reply:
ok, sorry for having missed the comment